R/gradethis_exercise_checker.R

```diff
@@ -169,7 +169,7 @@ check_exercise <- function(
   )
 
   # Skip parse checking if it was already done in {learnr}
-  if (!learnr_includes_parse_check(stage)) {
+  if (!learnr_includes_parse_check(stage) && identical(learnr_args$engine %||% "r", "r")) {
```

Comment:
In the future, I'll think we'll want to standardize how we discover what language a piece of code is, but this is fine for now